frameworks/Groovy/grails/setup.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
-
 sed -i 's|jdbc:mysql://.*:3306|jdbc:mysql://'"${DBHOST}"':3306|g' hello/grails-app/conf/DataSource.groovy
 
+export RESIN_HOME=${IROOT}/resin-4.0.41
```

Comment:
Typo here - forgot to add in the line `export JAVA_HOME=/usr/lib/jvm/java-1.7.0-openjdk-amd64`